src/checker.cpp

```diff
@@ -973,6 +973,8 @@ gb_internal void init_universal(void) {
 	add_global_string_constant("ODIN_ROOT",    bc->ODIN_ROOT);
 	add_global_string_constant("ODIN_BUILD_PROJECT_NAME", bc->ODIN_BUILD_PROJECT_NAME);
 
+	add_global_bool_constant("WINDOWS_SUBSYSTEM", bc->use_subsystem_windows);
```

Comment:
It makes more sense to do this:
```
add_global_string_constant("ODIN_WINDOWS_SYBSYSTEM", the_string);
```

Reply:
This also makes it work on non-Windows systems by having it be an empty string constant

Reply:
Sensible.